mangle.json

```
@@ -0,0 +1,15 @@
+{
+  "minify": {
+    "mangle": {
+      "properties": {
+        "regex": "/^_/"
+      },
+      "toplevel": true
+    },
+    "compress": {
+      "reduce_funcs": false,
+      "passes": 2
```

No real reason to limit passes - Terser will stop processing once two subsequent passes result in no change.

```suggestion
      "passes": 10
```

---

Oh this is very interesting to know, thank you again!